.circleci/config.yml

```diff
@@ -72,7 +72,8 @@ jobs:
             # Unset trap
             trap - ERR
             # Create Singularity container remotely and push to cloud
-            singularity build --remote library://aiconslab/miracl/miracl:latest miracl.def
```

Comment:
The `/code` directory will have 755 permissions. `rwxr-xrr-x`. Is that fine or should I work on fixing that? 

If we need to fix it to 777, I'm assuming if we make that change in the Docker image it will carry through once we make the singularity image correct?